dotnet/src/libraries/AuthorizedStore.Fake/Stores/StoreDao.cs

```diff
@@ -61,6 +61,11 @@ public async Task<IPagedList<Store>> GetListAsync(StoreCriteria criteria)
             var result = string.IsNullOrWhiteSpace(criteria.Name)
                 ? stores
                 : stores.Where(c => c.Name.Contains(criteria.Name));
+            if (criteria.CategoryIds.Length > 0)
```

Comment:
```suggestion
            if (criteria.CategoryIds?.Length > 0)
```
若條件沒有 `CategoryIds` 參數，此處會有 null reference exception。